tests/e2e/instrumentation-go/01-install-app.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment-with-sidecar
+spec:
+  selector:
+    matchLabels:
+      app: my-pod-with-sidecar
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: my-pod-with-sidecar
+      annotations:
+        sidecar.opentelemetry.io/inject: "true"
+        instrumentation.opentelemetry.io/inject-go: "true"
+        instrumentation.opentelemetry.io/go-target-exec: /usr/local/bin/emojivoto-voting-svc
```

Comment:
Should this be targeting the `productcatalogservice` binary?

Reply:
Good catch.  Since the e2e test doesn't actually check the output of the instrumentation inject this wasn't caught (or tested), but best to use a proper value anyways.  Updated.